src/case_encoder.cc

```diff
@@ -0,0 +1,137 @@
+// Copyright 2016 Google Inc.
```

Comment:
Seems weird adding an old copyright header for a new code, but I don't have the solution.

Reply:
Meh :) We will figure that out if we ever share back into the original repository.